web-workout-tracker/workout/src/views/App.js

```diff
@@ -19,6 +20,7 @@ function App() {
       <Router>
         <Route exact path="/" component={Landing} />
         <Route exact path='/user-settings' component={UserSettings} />
+        <Route exact path='/onboarding/body-goal' component={BodyGoal} />
```

Comment:
looks good. I think eventually we need to make a private route to user settings and onboarding.